samples/dnn/virtual_try_on.py

```diff
@@ -0,0 +1,435 @@
+#!/usr/bin/env python
+'''
+You can download the Geometric Matching Module model from https://drive.google.com/file/d/1oBnM9R-LgH0APkwxOiEUbFqbKArh0ZcR/view?usp=sharing
+You can download the Try-On Module model from https://drive.google.com/file/d/1czKbtPlFzE3-CN7lHsUNsgvKXLVi244c/view?usp=sharing
+You can download the cloth segmentation model from https://www.dropbox.com/s/qag9vzambhhkvxr/lip_jppnet_384.pb?dl=0
+You can find the OpenPose proto in opencv_extra/testdata/dnn/openpose_pose_coco.prototxt
+and get .caffemodel using opencv_extra/testdata/dnn/download_models.py
+'''
+
+import argparse
+import numpy as np
+import cv2 as cv
+
+from numpy import linalg
+from common import findFile
+from human_parsing import parse_human
+
+backends = (cv.dnn.DNN_BACKEND_DEFAULT, cv.dnn.DNN_BACKEND_INFERENCE_ENGINE, cv.dnn.DNN_BACKEND_OPENCV)
+targets = (cv.dnn.DNN_TARGET_CPU, cv.dnn.DNN_TARGET_OPENCL, cv.dnn.DNN_TARGET_OPENCL_FP16, cv.dnn.DNN_TARGET_MYRIAD)
+
+parser = argparse.ArgumentParser(description='Use this script to run virtial try-on using CP-VTON',
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('--input_image', '-i', help='Path to image with person. Skip this argument to capture frames from a camera.')
+parser.add_argument('--input_cloth', '-c', required=True, help='Path to target cloth image')
+parser.add_argument('--gmm_model', '-gmm', default='gmm.onnx', help='Path to Geometric Matching Module .onnx model.')
+parser.add_argument('--tom_model', '-tom', default='tom.onnx', help='Path to Try-On Module .onnx model.')
+parser.add_argument('--segmentation_model', default='lip_jppnet_384.pb', help='Path to cloth segmentation .pb model.')
+parser.add_argument('--openpose_proto', required=True, help='Path to OpenPose .prototxt model was trained on COCO dataset.')
+parser.add_argument('--openpose_model', required=True, help='Path to OpenPose .caffemodel model was trained on COCO dataset.')
```

Comment:
Let's add default values for OpenPose model and try `findFile`

Reply:
FindFile used in main
```
 pose = get_pose_map(args.input_image, findFile(args.openpose_proto),
                        findFile(args.openpose_model), args.backend, args.target)
```

Reply:
Yeah, so please add default values as well so it can be detected automatically